### DISCUSSION TITLE
automatic change of brightness when on battery

### DISCUSSION THREAD

**Discussion by ndhorniak** [seerge/g-helper#3136] [Ideas]
Hello, I was thinking if it will be possible to add option to change "brightness when on battery".
Simply when you unplug your charging cable it will automatically set to the brightness that you set up in GHelper.

**Comment by seerge** [comment]
@ndhorniak hello, you kind of can do that already.

Windows remembers display brightness level for each separate windows **power plan**. By default G-Helper will always use "Balanced" power plan. 

But you can just make a copy of a balanced plan in windows and assign it to one of the modes (for example to Silent) [as explained here.](https://github.com/seerge/g-helper/wiki/Power-user-settings#custom-windows-power-plans-with-each-mode) Then when you would automatically switch to Silent on battery - it will use last user brightness for that plan.
